etna/insights/models.py

```diff
@@ -46,12 +59,29 @@ class InsightsPage(HeroImageMixin, TeaserImageMixin, BasePage):
 
     sub_heading = models.CharField(max_length=200, blank=False)
     body = StreamField(InsightsPageStreamBlock, blank=True, null=True)
-
+    topic = models.ForeignKey(
+        "collections.TopicExplorerPage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    time = models.ForeignKey(
```

Comment:
Maybe `time_period` would be a more obvious name for this field?

Reply:
Yes, changed now.